packages/database/src/migrations/20210426232045-AddStriveFacilityEpiCurveDashboard-modifies-data.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+import { insertObject } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+
+const REPORT = {
+  id: 'PG_Strive_PNG_Facility_EpiCurve',
+  dataBuilder: 'analyticsPerPeriod',
+  dataBuilderConfig: {
+    series: [
+      { key: 'QMAL +ve', dataElementCode: 'STR_QMAL05' },
+      { key: 'Pf +ve', dataElementCode: 'STR_PF05' },
+      { key: 'Pv +ve', dataElementCode: 'STR_PV05' },
+      { key: 'Pm +ve', dataElementCode: 'STR_PM05' },
+      { key: 'Po +ve', dataElementCode: 'STR_PO05' },
+    ],
+    aggregationType: 'SUM_EACH_WEEK',
+    entityAggregation: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+      aggregationType: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+    },
+  },
+  viewJson: {
+    name: 'Lab Confirmed Positive Results, Line Graph',
```

Comment:
```suggestion
    name: 'Lab Confirmed Positive Results',
```

Do you suggest we can remove this in report title?

Reply:
I might leave as is for now, as there is another dashboard that is a 'Lab Confirmed Positive Results' dashboard as well.

It's possible this dashboard ought to be called "Epi Curve" (I'm really not sure I think I'd need Erin's input) but this is the kind of thing that can easily be edited via the admin panel post-release 👍